kafkajobs/config.go

```diff
@@ -30,43 +33,62 @@ func (c *config) InitDefault() ([]kgo.Opt, error) {
 	opts = append(opts, kgo.MaxVersions(kversion.Stable()))
 	opts = append(opts, kgo.RetryTimeout(time.Minute*5))
 
+	if c.SSL != nil && c.SSL.Enabled {
```

Comment:
It should be enough to check for c.SSL for nil.

Reply:
Described in comment above.